testutil/e2e/sdk/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "e2e",
+  "version": "1.0.0",
+  "description": "Run using the following command",
+  "main": "sdk.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@lavanet/lava-sdk": "^0.13.1",
```

Comment:
I want to use lava-sdk from the local path (ecosystem/lava-sdk) instead of fetching it from npm so we can make failures a blocker. all you need it to branch out from the fork branch so lava-sdk will be there :)